.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+## PR Checklist
```

Comment:
```suggestion
## Pull Request Checklist
```

Believe it or not I have had a few times where folks didn't know what "PR" meant; we can just spell it out at the top which usually clarifies